qml/CustomLegend.qml

```diff
@@ -220,4 +225,16 @@ Rectangle {
     function removeSeries(seriesIndex) {
         seriesRemoved(seriesIndex);
     }
+
+    function getLabel(seriesIndex) {
+        return seriesModel.get(seriesIndex).seriesName
```

Comment:
I'd append at the end of the series label the data units. For example for the latency it'll be something like `mean-part1-part2[us]` .

Reply:
Refactor csv in order to have the following format:

| | <datakind> |
| | <chartbox_title> |
| ms | <data_unit> |
| PosixTime | <series_label> |
| time | data |